### PR TITLE
Some opinionated makefile changes.

### PR DESCRIPTION
**Pull request by mcpherrinm**
Bikeshed away.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -1,15 +1,20 @@
 # Find all non-vendor source files
 SOURCE_FILES = $(shell find . \( -name '*.go' -not -path './vendor*' \)) 
 
-depends:
-	glide -q install
-
-build:
+certigo: $(SOURCE_FILES)
```
Comment (mcpherrinm) [inline Makefile]: I could be convinced to leave this as `build:`

**Comment by csstaub** [comment]
Also need to change readme.

File: Makefile [hunk 2]
```diff
@@ -1,15 +1,20 @@
...
 	go test -v `glide novendor`
 
 check:
 	go vet -v `glide novendor`
 	!(gofmt -d $(SOURCE_FILES) | grep .)
+
+install: depends
+	go build -o $(GOPATH)/bin/certigo .
+
+.PHONY : default depends test check
```
Comment (csstaub) [inline Makefile]: I don't see a default target?

**Comment by csstaub** [comment]
@mcpherrinm It would be nice if we could use a unified Makefile for certigo/ghostunnel/keywhiz-fs.

File: Makefile [hunk 3]
```diff
@@ -1,15 +1,20 @@
...
+
 test:
 	go test -v `glide novendor`
 
 check:
 	go vet -v `glide novendor`
 	!(gofmt -d $(SOURCE_FILES) | grep .)
+
+install: depends
+	go build -o $(GOPATH)/bin/certigo .
```
Comment (csstaub) [inline Makefile]: You probably want `go install`, not `go build`.

**Pull request closed without merging**